src/ValidateEmail.js

```diff
@@ -0,0 +1,6 @@
+// Check if the email is valid
+const validateEmail = (email) => {
+  return email.match(/^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/);
+}
```

Comment:
This works perfectly well, just a heads up that it may make the most sense/be cleaner to put all of your validators in a folder called `utils` - that's the professional convention. Nice work though!